applications/fishing-map/src/features/app/App.tsx

```diff
@@ -125,7 +125,8 @@ function App(): React.ReactElement {
 
   return (
     <Fragment>
-      <MapboxRefProvider>
+      {/* Value as null as there is no needed to set a default value but Typescript complains */}
+      <MapContext.Provider value={null as any}>
```

Comment:
Is there any reason to not have the `Provider` at the top level, just below the Fragment?

Reply:
Yep, the Menu and the Modal components are "siblings" so we needed a Fragment parent component

Reply:
I mean get rid of Fragment and
```
<Provider>
  <SplitView />
  <Menu />
  <Modal />
  ....
</Provider>

Reply:
Oh I get it! sounds good, I've updated it in https://github.com/GlobalFishingWatch/frontend/commit/e809883567fb188f663c85a825c94bfe11e9df73